proto/viam/app/data/v1/data.proto

```diff
@@ -53,19 +53,22 @@ service DataService {
   rpc BoundingBoxLabelsByFilter(BoundingBoxLabelsByFilterRequest) returns (BoundingBoxLabelsByFilterResponse);
 }
 
+// Order specifies the order in which data is returns.
 enum Order {
   ORDER_UNSPECIFIED = 0;
   ORDER_DESCENDING = 1;
   ORDER_ASCENDING = 2;
 }
 
+// DataRequest encapsulates the filter for the data, a limit on the maximum results returned, a last string associated with the last returned document, and the sorting order by time.
 message DataRequest {
   Filter filter = 1;
   uint64 limit = 2;
   string last = 3;
   Order sort_order = 4;
 }
 
+// Filter defines the fields over which we can filter data using a logic AND.
```

Comment:
This could also benefit from an example, such as `if component_type and robot_id are specified, only data from that ``robot_id`` of type ``component_type`` is returned`.

Reply:
[supernit] would use a single \`, i was using two since didn't seem to work to escape it within the formatting